backend/uclapi/workspaces/management/commands/feed_occupeye_cache.py

```diff
@@ -9,8 +11,23 @@ class Command(BaseCommand):
     )
 
     def handle(self, *args, **options):
-        print("Running OccupEye Caching Operation")
-        print("[+] Feeding Cache")
-        cache = OccupeyeCache()
-        cache.feed_cache(full=True)
-        print("Done!")
+        try:
+            print("Running OccupEye Caching Operation")
+            print("[+] Feeding Cache")
+            cache = OccupeyeCache()
+            cache.feed_cache(full=True)
+            print("Done!")
+            incident_name = get_incident_name("Occupeye")
+            if incident_name is not None:

```

Comment:
```suggestion
            if incident_name:

```

Reply:
and maybe an `else`? If this slips by we wouldn't know... Maybe we should print something in this case...

Reply:
Can do, it will be kind of annoying since it will always print the else in a dev environment though. Added.